wrapper/src/main/java/software/amazon/jdbc/plugin/efm/HostMonitoringConnectionPlugin.java

```diff
@@ -116,8 +122,11 @@ public HostMonitoringConnectionPlugin(
     }
 
     this.pluginService = pluginService;
+    this.telemetryFactory = pluginService.getTelemetryFactory();
     this.properties = properties;
     this.monitorServiceSupplier = monitorServiceSupplier;
+
+    this.invalidatedConnectionCounter = telemetryFactory.createCounter("efm.connection.invalidated");
```

Comment:
Is this counter updated in a different file?

Reply:
Moved counter